static/templates/task/base.html

```diff
@@ -1,4 +1,4 @@
-<div class="_main-content" style="width: 100%">
```

Comment:
not in the browser: i guess you wanna keep `
+            <form ng-init="template.upload = task.upload"
`
but you have to merge correctly with dev2